.github/CODEOWNERS

```diff
@@ -2,7 +2,6 @@
 * @ethereum-optimism/go-reviewers
```

Comment:
It seems there are promoted errors here

Reply:
This file is the same as upstream: `git diff op-merge develop .github/CODEOWNERS` shows no diff. Looks like this file is used to automatically assign reviewers, we're doing this manually so it doesn't matter.